### ISSUE TITLE
adapt-contrib-mcq/gmcq: field inputs not associated with question

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#3142]
### Subject of the issue
When tabbing through the inputs they should be associated with the question text.
As seen in these tests [with fieldset](https://www.powermapper.com/tests/screen-readers/labelling/fieldset-without-yes-no-radio-buttons/), [without fieldset](https://www.powermapper.com/tests/screen-readers/labelling/fieldset-with-yes-no-radio-buttons/).

In adapt we currently use `role=group` instead of the `fieldset` element as set out in [aria17](https://www.w3.org/TR/WCAG20-TECHS/ARIA17.html), rather than [h71](https://www.w3.org/TR/WCAG20-TECHS/H71.html) however, we should possibly also include an `aria-label` or an `aria-labelledby` to mimic the `legend` element.

[blank legend](https://www.powermapper.com/tests/screen-readers/labelling/fieldset-blank-legend/), [no legend](https://www.powermapper.com/tests/screen-readers/labelling/fieldset-no-legend/).
